dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jakarta.persistence-3.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jakarta.persistence-3.0
+singleton=true
+IBM-Process-Types: server, \
+ client
+-features=com.ibm.websphere.appserver.jakarta.persistence.base-3.0
+-bundles=com.ibm.ws.jakartaee.persistence.api.3.0
+-jars=com.ibm.websphere.jakartaee.persistence.3.0; location:=dev/api/spec/; mavenCoordinates="jakarta.persistence:jakarta.persistence-api:3.0"
+kind=beta
```

Comment:
All EE 9 features need to be `kind=noship` and `edition=full` so we don't accidentally ship them before they are ready